### DISCUSSION TITLE
How to add "Informartions" events policy in RETENTION

### DISCUSSION THREAD

**Discussion by zhujiancc** [datalust/seq-tickets#1313] [Help]
I add "Informartions" signal to filter info events 
![image](https://user-images.githubusercontent.com/19665595/132281105-1798542b-bec8-4a03-8e56-274fac11e07b.png)

But  I can not set retention policy for this new signal event 
![image](https://user-images.githubusercontent.com/19665595/132281162-196158c8-44ad-4601-acfa-6a93f7f4dabe.png)

So is it any way to make it

**Comment by nblumhardt** [comment]
Hi @zhujiancc!

The problem here will be that Informations is a personal signal, while retention policies can only use shared signals.

You can fix this by editing the Informations signal, clicking the drop-down next to the signal title, and choosing "Shared" before saving it.

I'll leave this open, I think we can improve the UI here by:

 * Showing all signals (personal and shared) in the retention policy editor drop-down, and
 * Presenting an informative error message when trying to save the resulting policy, if personal signals are chosen.

Let me know if you need any more info.

Best regards,
Nick
